app/model/Scheme.scala

```diff
@@ -19,7 +19,9 @@ package model
 import play.api.libs.json._
 import reactivemongo.bson.{ BSON, BSONHandler, BSONString }
 
-case class SchemeId(value: String)
+case class SchemeId(value: String) {
+  implicit override def toString: String = value
```

Comment:
I've never seen `implicit` for `toString`, could you please clarify why it's needed?